tests/formulas/irpp_cehr.yaml

```diff
@@ -108,3 +108,115 @@
     salaire_imposable: 1500000
   output:
     irpp: -697262
+
+- name: CEHR - Célibataire ayant un revenu fiscal de référence de 200 000 €
+  period: 2019
+  input:
+    individus:
+      individu1:
+        age: 50
+    foyer_fiscal:
+      declarants: [individu1]
+      rfr: 200_000
+  output:
+    cehr: 0
+
+- name: CEHR - Célibataire ayant un revenu fiscal de référence de 800 000 €
+  period: 2019
+  input:
+    individus:
+      individu1:
+        age: 50
+    foyer_fiscal:
+      declarants: [individu1]
+      rfr: 800_000
+  output:
+    cehr: 19_500
+
+- name: CEHR - Couple soumis à une imposition commune ayant un revenu fiscal de référence de 400 000 €
+  period: 2019
+  input:
+    individus:
+      individu1:
+        age: 50
+      individu2:
+        age: 55
+    foyer_fiscal:
+      declarants: [individu1, individu2]
+      rfr: 400_000
+  output:
+    cehr: 0
+
+- name: CEHR - Couple soumis à une imposition commune ayant un revenu fiscal de référence de 1 500 000 €
+  period: 2019
+  input:
+    individus:
+      individu1:
+        age: 50
+      individu2:
+        age: 55
+    foyer_fiscal:
+      declarants: [individu1, individu2]
+      rfr: 1_500_000
+  output:
+    cehr: 35_000
+
+- name: |
+    CEHR - Couple soumis à une imposition commune ayant un revenu fiscal de référence N de 800 000 €,
+    un RFR n-1 de 350 000 € et un RFR n-2 de 400 000 € (test du lissage)
+  period: 2019
+  input:
+    individus:
+      individu1:
+        age: 50
+      individu2:
+        age: 55
+    foyer_fiscal:
+      declarants: [individu1, individu2]
+      rfr:
+        2019: 800_000
+        2018: 350_000
+        2017: 400_000
+      calcul_lissage_cehr: true
+  output:
+    cehr: 5_250
+
+- name: |
+    CEHR - Couple soumis à une imposition commune ayant un revenu fiscal de référence N de 800 000 €,
+    un RFR n-1 de 350 000 € et un RFR n-2 de 400 000 € (mais calcul du lissage désactivé)
+  period: 2019
+  input:
+    individus:
+      individu1:
+        age: 50
+      individu2:
+        age: 55
+    foyer_fiscal:
+      declarants: [individu1, individu2]
+      rfr:
+        2019: 800_000
+        2018: 350_000
+        2017: 400_000
+      calcul_lissage_cehr: false
+  output:
+    cehr: 9_000
+
```

Comment:
Ajouter également deux tests : un quand `condition_2` False et un autre quand `condition_3` False

Reply:
Ok, noté !